Editor/LocalisationEditor.cs

```diff
@@ -15,10 +14,11 @@ public class LocalisationEditor : EditorWindow
 		/// </summary>
 		public const int KEY_CRC_ENCODING_VERSION = 1;
 
+		// Code generation
```

Comment:
I don't know what this comment means. Are these generated values? Are they related to some code generation?

Reply:
Updated